src/js/controller/controller.js

```diff
@@ -899,7 +871,9 @@ define([
             // Add commands from CoreLoader to queue
             apiQueue.queue.push.apply(apiQueue.queue, commandQueue);
 
-            _setup.start();
+            _setup.start().then(_playerReady).catch((err) => {
```

Comment:
I think you can shorthand this as ` _setup.start().then(_playerReady).catch(setupError)`